contrib/lkt/language/prelude.lkt

```diff
@@ -54,6 +55,10 @@ struct ASTList implements Sized, Indexable[T], Iterator[T] {
 @builtin class Equation {
 }
 
+@builtin struct PropertyError {
+    exception_message : String = "PropertyError exception"
```

Comment:
um... doesn't stick to new coding style I think?  Should be `exception_message:` without space, right?